data-structures-queues/is-empty.js

```diff
@@ -0,0 +1,5 @@
+export default function isEmpty(queue) {
+  if (queue.peek() === undefined) {
+    return true;
+  } else return false;
```

Comment:
Good, but more succinctly, `return queue.peek() === undefined;`
(It's not generally a good idea to write `if (x) return true; else return false;` since you can just write `return x;`.)

Reply:
Oh, okay.  Studying that.